internal/controllers/server/tests/import-error/00-prerequisites.yaml

```diff
@@ -0,0 +1,49 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - command: kubectl create secret generic openstack-clouds --from-file=clouds.yaml=${E2E_KUTTL_OSCLOUDS} ${E2E_KUTTL_CACERT_OPT}
+    namespaced: true
+  - script: cat ../templates/create-flavor.tmpl | envsubst | kubectl -n ${NAMESPACE} apply -f -
+---
+apiVersion: openstack.k-orc.cloud/v1alpha1
+kind: Image
+metadata:
+  name: import-error-external
+spec:
+  cloudCredentialsRef:
+    cloudName: openstack-admin
+    secretName: openstack-clouds
+  managementPolicy: managed
+  resource:
+    content:
+      diskFormat: qcow2
+      download:
+        url: https://download.cirros-cloud.net/0.6.3/cirros-0.6.3-x86_64-disk.img
```

Comment:
I think we're going to piss them off downloading this for every server test in every test run. We're at risk of being blocked. I think this is a critical issue, in fact, and I don't think we should merge this change until we've fixed it.

For the Image download tests we start a local http server which serves the image files. I wonder if we should do that here too. Lets pull the cirros image down as an artifact and serve it from localhost.

Reply:
Sure, we can change it. I thought it was common in OpenStack to download a cirros image from their servers for CI needs.

If we save the image in our git, we can pull from github like I did in the image tests.

https://github.com/k-orc/openstack-resource-controller/blob/7a22deab8b9932586a240beb2c2a84c89991c416/internal/controllers/image/tests/create-full/00-image.yaml#L37

Reply:
@mdbooth I think you only want to download from a remote server once and then use a local copy for all the tests, right?

Reply:
It should now pull the image from github.